src/Classes/Canvas.ts

```diff
@@ -117,8 +118,27 @@ export class Canvas {
      * Paints the canvas background
      */
     public paint(): void {
+        const container = this.container;
+        const options = container.options;
+
         if (this.context) {
-            this.context.fillStyle = "rgba(255, 255, 255, 0)";
+            if (options.backgroundMask.enable && options.backgroundMask.cover) {
+                const color = Utils.getParticleColor(options.backgroundMask.cover);
+
+                if (color) {
+                    this.paintBase(Utils.getStyleFromColor(color));
+                } else {
+                    this.paintBase();
+                }
+            } else {
+                this.paintBase();
+            }
+        }
+    }
+
+    private paintBase(baseColor = "rgba(255, 255, 255, 0)"): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'baseColor' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5164124)